sdk/identity/Azure.Identity/src/EnvironmentCredential.cs

```diff
@@ -66,9 +66,9 @@ internal EnvironmentCredential(CredentialPipeline pipeline)
                 {
                     _credential = new ClientSecretCredential(tenantId, clientId, clientSecret, _pipeline);
                 }
-                else if (username != null && password != null && tenantId != null && clientId != null)
+                else if (username != null && password != null)
                 {
-                    _credential = new UsernamePasswordCredential(username, password, clientId, tenantId, _pipeline);
+                    _credential = new UsernamePasswordCredential(username, password,  tenantId, clientId,_pipeline);
```

Comment:
NIT: Some minor spacing issues here (two spaces between the comma after `password` and before `tenantId` and no spaces between `clientId` and `_pipeline`).

Reply:
This is now resolved with the latest push